pilot/pkg/xds/ads.go

```diff
@@ -880,6 +879,7 @@ func (s *DiscoveryServer) addCon(conID string, con *Connection) {
 	s.adsClientsMutex.Lock()
 	defer s.adsClientsMutex.Unlock()
 	s.adsClients[conID] = con
+	recordXDSClients(con.proxy.Metadata.IstioVersion, 1)
```

Comment:
@howardjohn is `IstioVersion` definitely computed at this point?

Reply:
Yes